tests/unit/constraints/test_tabular.py

```diff
@@ -765,7 +765,7 @@ def test_transform(self):
         try:
             [uuid.UUID(u) for c, u in out['b#c'].items()]
         except ValueError:
-            pytest.fail('ValueError')
+            pytest.fail('ValueError')  # noqa: PT009
```

Comment:
`flake8-pytest-style` update doesn't let you to use these anymore

Reply:
Why does this need to be in a try/except block? Wouldn't the `ValueError` cause the test to fail anyways?

Reply:
Does this line usually get hit? I'm not sure why we need this anyway

Reply:
@frances-h I believe it's for clarity: it's weird to just create a list and not use it. This makes it clear we are expecting `uuid.UUID()` to not raise a ValueError.

@amontanez24 It never gets reached, since the test (hopefully) never fails. The list itself should stay there though, since we need to verify that every value was correctly converted into an id.

I really don't think this matters though, I'm just deleting it as you suggested so we don't have to add the `noqa`.

Reply:
@fealho re: clarity, another option could be to add an `assert` before the list. Assuming the list is not empty, it'll always pass and maybe be clearer that we want to explicitly test creating a list of UUIDs.